code-list-scripts/ashe.yaml

```diff
@@ -34,3 +37,6 @@ imports:
       - ../geo-specific-codelist/gremlin/travel-to-work-geography.grm
       - ../geo-specific-codelist/local-enterprise-partnership-geography.grm
       - gremlin/sic.grm
+      - cypher/soc.grm
```

Comment:
Im guessing this path should be `gremlin/...`

Reply:
yes you're right, always seem to miss these